src/lua/lua_script.rs

```diff
@@ -84,6 +84,10 @@ fn create_default_lua_state(
             .load(include_str!("../../resources/lua/tasks.lua"))
             .call::<_, rlua::Value>(())?;
         globals.set("tasks", lua_tasks)?;
+        let lua_triggers = ctx
+            .load(include_str!("../../resources/lua/trigger.lua"))
+            .call::<_, rlua::Value>(())?;
+        globals.set("trigger", lua_triggers)?;
```

Comment:
The doc doesn't match this naming now. 😋

Reply:
oops